test/e2e/infra/vagrant/Vagrantfile

```diff
@@ -23,11 +23,16 @@ K8S_SERVICE_NETWORK_CIDR = (MODE == "v4") ? K8S_SERVICE_NETWORK_V4_CIDR : K8S_SE
 NODE_NETWORK_V4_PREFIX = "192.168.77."
 NODE_NETWORK_V6_PREFIX = "fd3b:fcf5:3e92:d732::"
 
+MEMORY = 2048
+if ENV['LARGE'] == "true"
```

Comment:
can you use a more specific name for the env variable, e.g. `K8S_NODE_LARGE`

Reply:
done